helpers/announce.js

```diff
@@ -0,0 +1,41 @@
+import { offscreenStyles } from '../components/offscreen/offscreen-styles.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class AnnounceContainer extends LitElement {
+
+	static get styles() {
+		return [ offscreenStyles ];
+	}
+
+	constructor() {
+		super();
+		this._messages = [];
+	}
+
+	announce(text) {
+		this._messages.push(text);
+		this.requestUpdate();
+	}
+
+	render() {
+		return html`<div aria-live="polite" class="d2l-offscreen">
+			${this._messages.map(message => html`<div>${message}</div>`)}
+		</div>`;
+	}
+
+}
+
+customElements.define('d2l-announce-container', AnnounceContainer);
+
+let container;
+
+export function announce(text) {
+	if (!text) return;
+	if (!container) {
+		container = document.createElement('d2l-announce-container');
+		document.body.appendChild(container);
```

Comment:
This appends the container to the `body`, which is different from `iron-a11y-announcer`, which appends to the `head`.  Unfortunately, when appending to the `head`, no announcements are made with Safari+VO.  My guess is that their reason for appending to the `head` was an attempt to avoid reaching the messages with the virtual cursor. 

A user who discovers these messages using the virtual cursor may be somewhat confused.  One option to provide context for these messages could be to simply include a heading/label that reads "recent messages".

For what it's worth, our MVC `D2L.LP.Web.UI.Html.Aria.LogMessage` appends to the `body` and is therefore visible to the virtual cursor.